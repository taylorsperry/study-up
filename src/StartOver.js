import React from 'react';

const StartOver = props => {
    return (
        <button className="result-button" onClick={props.reset}>
            Start Over
        </button>
        
    );
};

export default StartOver;