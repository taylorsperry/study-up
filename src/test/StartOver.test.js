import React from "react";
import StartOver from "../StartOver";
import { shallow } from "enzyme"

describe("StartOver", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<StartOver />)
  })

  it("should match snapshot when all data is passed correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
})