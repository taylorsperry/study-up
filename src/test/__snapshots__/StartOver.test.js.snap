// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartOver should match snapshot when all data is passed correctly 1`] = `
<button
  className="result-button"
>
  Start Over
</button>
`;
